android/src/main/java/com/applovin/reactnative/AppLovinMAXModule.java

```diff
@@ -92,10 +92,14 @@
     private int           lastRotation;
 
     // Store these values if pub attempts to set it before initializing
-    private String       userIdToSet;
-    private List<String> testDeviceAdvertisingIdsToSet;
-    private Boolean      verboseLoggingToSet;
-    private Boolean      creativeDebuggerEnabledToSet;
+    private       String              userIdToSet;
+    private       String              userSegmentNameToSet;
```

Comment:
On second thought - we're deprecating user user segment actually:

https://app.asana.com/0/337926977135969/1202550757307961

Reply:
Ok, removed user segment.  Will request to remove it from the doc.   Also, my bad.  I missed locationCollection update in Android.    Tested again with both Android and iOS.